README.md

```diff
@@ -807,8 +807,11 @@ OK:
 TODO-A: - Ubuntu does not carry a delta
 TODO-B: - Ubuntu does carry a delta, but it is reasonable and maintenance under
 TODO-B:   control
-TODO: - symbols tracking is in place
-TODO: - symbols tracking not applicable for this kind of code.
+TODO-A: - symbols tracking is in place.
+TODO-B: - For c++ libraries - symbols tracking isn't in place but the owning
+TODO-B:   team tried to set it up and came back with a reasonable rational
+TODO-B:   of why it isn't practical to do for the package.
```

Comment:
It could boost the MIR team confidence if some other means of ABI compat tooling is in place, e.g. using abigail or abi-compliance-check in (upstream?) CI, bumping SOVER with every package update.

Reply:
yeah, whenever simple tracking failed giving this a look is absolutely worth it.
After all, this isn't to pacify a process - but to help all ubuntu developers and users by ensuring quality.

Would you mind adding a statement about it?

Reply:
We mention something about https://wiki.ubuntu.com/DailyRelease/FAQ down below, maybe that should be extended accordingly.